2024-12-30 04:26:14,212 - ERROR - URL Error https://59f1cbe63db89.streamlock.net:1443/rsnewssports/_definst_/rsnewssports/playlist.m3u8: HTTPSConnectionPool(host='59f1cbe63db89.streamlock.net', port=1443): Max retries exceeded with url: /rsnewssports/_definst_/rsnewssports/playlist.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for '59f1cbe63db89.streamlock.net'. (_ssl.c:1000)")))
2024-12-30 04:26:35,906 - ERROR - URL Error https://tv.vidatel.com.br/hls/tvloadinghd/live.m3u8: HTTPSConnectionPool(host='tv.vidatel.com.br', port=443): Max retries exceeded with url: /hls/tvloadinghd/live.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f20872b45f0>, 'Connection to tv.vidatel.com.br timed out. (connect timeout=15)'))
2024-12-30 04:26:47,711 - ERROR - URL Error https://live.uniaoplus.com/hls/8b43MluBjzR7M1Uk.m3u8: HTTPSConnectionPool(host='live.uniaoplus.com', port=443): Max retries exceeded with url: /hls/8b43MluBjzR7M1Uk.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2087263650>: Failed to resolve 'live.uniaoplus.com' ([Errno -2] Name or service not known)"))
2024-12-30 04:26:48,675 - ERROR - URL Error https://59f1cbe63db89.streamlock.net:1443/interativafm/_definst_/interativafm/chunklist_w37160837.m3u8: HTTPSConnectionPool(host='59f1cbe63db89.streamlock.net', port=1443): Max retries exceeded with url: /interativafm/_definst_/interativafm/chunklist_w37160837.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for '59f1cbe63db89.streamlock.net'. (_ssl.c:1000)")))
2024-12-30 04:27:09,124 - ERROR - URL Error http://evpp.mm.uol.com.br:1935/band_live/terraviva/playlist.m3u8: HTTPConnectionPool(host='evpp.mm.uol.com.br', port=1935): Max retries exceeded with url: /band_live/terraviva/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f20872b46e0>, 'Connection to evpp.mm.uol.com.br timed out. (connect timeout=15)'))
2024-12-30 04:27:13,884 - ERROR - URL Error https://agrobrasiltv.easynet.com.br/memfs/e1f7edaa-38b1-4a53-9f58-9068d06972a2.m3u8: HTTPSConnectionPool(host='agrobrasiltv.easynet.com.br', port=443): Max retries exceeded with url: /memfs/e1f7edaa-38b1-4a53-9f58-9068d06972a2.m3u8 (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1000)')))
2024-12-30 04:27:36,476 - ERROR - URL Error https://streaming.almg.gov.br/live/tvalmg.m3u8: HTTPSConnectionPool(host='streaming.almg.gov.br', port=443): Max retries exceeded with url: /live/tvalmg.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2087261610>, 'Connection to streaming.almg.gov.br timed out. (connect timeout=15)'))
2024-12-30 04:28:01,375 - ERROR - URL Error https://www.portalcultura.com.br/playerhtml/funtelpa/tv_funtelpa/playlist.m3u8: HTTPSConnectionPool(host='www.portalcultura.com.br', port=443): Max retries exceeded with url: /playerhtml/funtelpa/tv_funtelpa/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2087263e60>, 'Connection to www.portalcultura.com.br timed out. (connect timeout=15)'))
2024-12-30 04:28:31,815 - ERROR - URL Error https://stream.nihonwebtv.com:3778/stream/play.m3u8: HTTPSConnectionPool(host='stream.nihonwebtv.com', port=3778): Max retries exceeded with url: /stream/play.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f20872b6fc0>, 'Connection to stream.nihonwebtv.com timed out. (connect timeout=15)'))
2024-12-30 04:28:50,014 - ERROR - URL Error http://evpp.mm.uol.com.br:1935/bandpracas/debatemg/playlist.m3u8: HTTPConnectionPool(host='evpp.mm.uol.com.br', port=1935): Max retries exceeded with url: /bandpracas/debatemg/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f20872b4650>, 'Connection to evpp.mm.uol.com.br timed out. (connect timeout=15)'))
2024-12-30 04:28:50,284 - ERROR - URL Error https://acesso.ecast.site:3186/live/esflixlive.m3u8: HTTPSConnectionPool(host='acesso.ecast.site', port=3186): Max retries exceeded with url: /live/esflixlive.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f20872b5dc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-30 04:29:02,764 - ERROR - URL Error https://d3rhmbr9yq9ljg.cloudfront.net/out/v1/4dab2f11ddef44f1ad6e39df6f31d67b/index.mpd: HTTPSConnectionPool(host='d3rhmbr9yq9ljg.cloudfront.net', port=443): Max retries exceeded with url: /out/v1/4dab2f11ddef44f1ad6e39df6f31d67b/index.mpd (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2087262cf0>: Failed to resolve 'd3rhmbr9yq9ljg.cloudfront.net' ([Errno -5] No address associated with hostname)"))
2024-12-30 04:29:17,779 - ERROR - URL Error http://150.161.93.220/hls/index.m3u8: HTTPConnectionPool(host='150.161.93.220', port=80): Max retries exceeded with url: /hls/index.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f20872b4530>, 'Connection to 150.161.93.220 timed out. (connect timeout=15)'))
2024-12-30 04:29:20,460 - ERROR - URL Error https://d10tspkqgju994.cloudfront.net/out/v1/7d01acad7c474d559689a421c90168db/index.mpd: HTTPSConnectionPool(host='d10tspkqgju994.cloudfront.net', port=443): Max retries exceeded with url: /out/v1/7d01acad7c474d559689a421c90168db/index.mpd (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f20872b7830>: Failed to resolve 'd10tspkqgju994.cloudfront.net' ([Errno -5] No address associated with hostname)"))
2024-12-30 04:29:21,074 - ERROR - URL Error https://stream.ichibantv.com:3874/hybrid/play.m3u8: HTTPSConnectionPool(host='stream.ichibantv.com', port=3874): Max retries exceeded with url: /hybrid/play.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f20872b5190>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-30 04:29:21,660 - ERROR - URL Error https://live.uniaoplus.com/hls/0gmf9QNc7bQ6V0Lu.m3u8: HTTPSConnectionPool(host='live.uniaoplus.com', port=443): Max retries exceeded with url: /hls/0gmf9QNc7bQ6V0Lu.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f20872b44a0>: Failed to resolve 'live.uniaoplus.com' ([Errno -2] Name or service not known)"))
2024-12-30 04:29:25,642 - ERROR - URL Error https://59f1cbe63db89.streamlock.net:1443/tvpampa/_definst_/tvpampa/playlist.m3u8: HTTPSConnectionPool(host='59f1cbe63db89.streamlock.net', port=1443): Max retries exceeded with url: /tvpampa/_definst_/tvpampa/playlist.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for '59f1cbe63db89.streamlock.net'. (_ssl.c:1000)")))
2024-12-30 04:29:27,883 - ERROR - URL Error https://acesso.ecast.site:3456/stream/play.m3u8: HTTPSConnectionPool(host='acesso.ecast.site', port=3456): Max retries exceeded with url: /stream/play.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2087263a70>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-30 04:29:45,145 - ERROR - URL Error https://5e837408ea907.streamlock.net:1936/tvci/tvci/playlist.m3u8: HTTPSConnectionPool(host='5e837408ea907.streamlock.net', port=1936): Max retries exceeded with url: /tvci/tvci/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f20872b4c20>, 'Connection to 5e837408ea907.streamlock.net timed out. (connect timeout=15)'))
2024-12-30 04:29:45,846 - ERROR - URL Error https://59f1cbe63db89.streamlock.net:1443/rsnews/_definst_/rsnews/playlist.m3u8: HTTPSConnectionPool(host='59f1cbe63db89.streamlock.net', port=1443): Max retries exceeded with url: /rsnews/_definst_/rsnews/playlist.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for '59f1cbe63db89.streamlock.net'. (_ssl.c:1000)")))
2024-12-30 04:29:45,993 - ERROR - URL Error https://59f1cbe63db89.streamlock.net:1443/rsnews03/_definst_/rsnews03/playlist.m3u8: HTTPSConnectionPool(host='59f1cbe63db89.streamlock.net', port=1443): Max retries exceeded with url: /rsnews03/_definst_/rsnews03/playlist.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for '59f1cbe63db89.streamlock.net'. (_ssl.c:1000)")))
2024-12-30 04:29:46,141 - ERROR - URL Error https://59f1cbe63db89.streamlock.net:1443/rsnews04/_definst_/rsnews04/playlist.m3u8: HTTPSConnectionPool(host='59f1cbe63db89.streamlock.net', port=1443): Max retries exceeded with url: /rsnews04/_definst_/rsnews04/playlist.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for '59f1cbe63db89.streamlock.net'. (_ssl.c:1000)")))
2024-12-30 04:29:46,295 - ERROR - URL Error https://59f1cbe63db89.streamlock.net:1443/danielteste/_definst_/danielteste/playlist.m3u8: HTTPSConnectionPool(host='59f1cbe63db89.streamlock.net', port=1443): Max retries exceeded with url: /danielteste/_definst_/danielteste/playlist.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for '59f1cbe63db89.streamlock.net'. (_ssl.c:1000)")))
2024-12-30 04:30:08,094 - ERROR - URL Error https://evpp.mm.uol.com.br/ne10/radiojornal-caruaru-avra-video-web.sdp/playlist.m3u8: HTTPSConnectionPool(host='evpp.mm.uol.com.br', port=443): Max retries exceeded with url: /ne10/radiojornal-caruaru-avra-video-web.sdp/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f20872b5d90>, 'Connection to evpp.mm.uol.com.br timed out. (connect timeout=15)'))
2024-12-30 04:30:08,278 - ERROR - URL Error https://streaming03.zas.media:1936/mixbelem/mixbelem/playlist.m3u8: HTTPSConnectionPool(host='streaming03.zas.media', port=1936): Max retries exceeded with url: /mixbelem/mixbelem/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f20872b5100>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-30 04:30:10,410 - ERROR - URL Error https://d1rkdnnqmg7q8z.cloudfront.net/out/v1/e6bb851f89aa42bf8fff9eb480ba667d/index.mpd: HTTPSConnectionPool(host='d1rkdnnqmg7q8z.cloudfront.net', port=443): Max retries exceeded with url: /out/v1/e6bb851f89aa42bf8fff9eb480ba667d/index.mpd (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2087263170>: Failed to resolve 'd1rkdnnqmg7q8z.cloudfront.net' ([Errno -5] No address associated with hostname)"))
2024-12-30 04:30:16,193 - ERROR - URL Error https://cdn-cdn-massafmsp.ciclano.io:1443/cdn-massafmsp/cdn-massafmsp/playlist.m3u8: HTTPSConnectionPool(host='cdn-cdn-massafmsp.ciclano.io', port=1443): Max retries exceeded with url: /cdn-massafmsp/cdn-massafmsp/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2087262de0>: Failed to resolve 'cdn-cdn-massafmsp.ciclano.io' ([Errno -2] Name or service not known)"))
2024-12-30 04:30:36,248 - ERROR - URL Error https://www.portalcultura.com.br/playerhtml/funtelpa/radio_funtelpa/chunklist_w1230197756.m3u8: HTTPSConnectionPool(host='www.portalcultura.com.br', port=443): Max retries exceeded with url: /playerhtml/funtelpa/radio_funtelpa/chunklist_w1230197756.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f208707d0d0>, 'Connection to www.portalcultura.com.br timed out. (connect timeout=15)'))
2024-12-30 04:30:38,379 - ERROR - URL Error https://cdn-mirtes787.ciclano.io:1443/mirtes787/mirtes787/playlist.m3u8: HTTPSConnectionPool(host='cdn-mirtes787.ciclano.io', port=1443): Max retries exceeded with url: /mirtes787/mirtes787/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2087263740>: Failed to resolve 'cdn-mirtes787.ciclano.io' ([Errno -2] Name or service not known)"))
2024-12-30 04:30:56,904 - ERROR - URL Error https://evpp.mm.uol.com.br/cultura2/cultura2.sdp/chunklist_w1456486161.m3u8: HTTPSConnectionPool(host='evpp.mm.uol.com.br', port=443): Max retries exceeded with url: /cultura2/cultura2.sdp/chunklist_w1456486161.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f20872b7560>, 'Connection to evpp.mm.uol.com.br timed out. (connect timeout=15)'))
2024-12-30 04:30:57,106 - ERROR - URL Error https://stream-04.nyc.dailymotion.com/sec(RiWEtJG0nU_F5iuiUOWAl8igtlYU8dfBa92nZpm-VlojN3kDIBruobNma11w-Q_V)/dm/3/x7wjrui/s/live-1.m3u8#cell=lnyc: HTTPSConnectionPool(host='stream-04.nyc.dailymotion.com', port=443): Max retries exceeded with url: /sec(RiWEtJG0nU_F5iuiUOWAl8igtlYU8dfBa92nZpm-VlojN3kDIBruobNma11w-Q_V)/dm/3/x7wjrui/s/live-1.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f20872b59a0>: Failed to resolve 'stream-04.nyc.dailymotion.com' ([Errno -2] Name or service not known)"))
2024-12-30 04:30:58,663 - ERROR - URL Error https:\\stm1.radiojhero.com:18008\: Invalid URL 'https:\\\\stm1.radiojhero.com:18008\\': No host supplied
2024-12-30 04:31:00,162 - ERROR - URL Error https://r10.ciclano.io:15075/stream?1651714823719: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-30 04:31:15,178 - ERROR - URL Error http://150.161.93.221/hls/stream/index.m3u8: HTTPConnectionPool(host='150.161.93.221', port=80): Max retries exceeded with url: /hls/stream/index.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f208707d2b0>, 'Connection to 150.161.93.221 timed out. (connect timeout=15)'))
2024-12-30 04:31:17,952 - ERROR - URL Error https://d10tspkqgju994.cloudfront.net/out/v1/7d01acad7c474d559689a421c90168db/index.mpd: HTTPSConnectionPool(host='d10tspkqgju994.cloudfront.net', port=443): Max retries exceeded with url: /out/v1/7d01acad7c474d559689a421c90168db/index.mpd (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f208707d5e0>: Failed to resolve 'd10tspkqgju994.cloudfront.net' ([Errno -5] No address associated with hostname)"))
